source/codegen/metadata/nirfmxvna/config.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 config = {
-    'api_version': '23.0.0',
```

Comment:
Did you want to set the code_readiness to Release down below?

'code_readiness': 'NextRelease' -> `code_readiness': 'Release'

And if so, please update the supported drivers table in the README:  https://github.com/ni/grpc-device?tab=readme-ov-file#supported-ni-drivers

Reply:
Corrected it to "Release" and update the README too. 
I have generated the files after building the gRPC component. Please let me know if any further changes are required
@reckenro